packages/gatsby/src/utils/__tests__/jobs-manager.js

```diff
@@ -334,4 +337,144 @@ describe(`Jobs manager`, () => {
       expect(isJobStale({ inputPaths })).toBe(false)
     })
   })
+
+  describe(`IPC jobs`, () => {
+    let listeners = []
+    beforeAll(() => {
+      jest.useFakeTimers()
+    })
+
+    beforeEach(() => {
+      process.env.ENABLE_GATSBY_EXTERNAL_JOBS = `true`
+      listeners = []
+      process.on = (type, cb) => {
+        listeners.push(cb)
+      }
```

Comment:
do we need to restore previous `process.on` in afterAll? would it have any impact on other tests if we didn't?

Reply:
it might :) if we add extra tests to this file